hoomd/filter/set_.py

```diff
@@ -0,0 +1,35 @@
+from hoomd.filter.filter_ import ParticleFilter
+from hoomd import _hoomd
+
+
+class ParticleFilterSetOperations(ParticleFilter):
```

Comment:
1. Should this be abstract? You can't instantiate it without `self._cpp_cls_name`.
2. Are users supposed to be able to subclass this? If not, make it private.
3. What about a set symmetric difference?

Reply:
1. See above comment.
2. I do need to make it private.
3. The set symmetric difference can be achieved with `SetDifference(Union(a, b), Intersection(a, b))` though I could see an argument for performance reasons.